src/planned-tests.html

```diff
@@ -39,34 +39,17 @@ <h1 class="govuk-heading-xl">
         {{ pageTitle }}
       </h1>
       <h2 class="govuk-heading-m govuk-!-margin-bottom-2">
-        Tuesday 25 May 2021
+        Tuesday 15 June 2021
```

Comment:
Do we need to change this [here](https://github.com/alphagov/notifications-govuk-alerts/blob/3eca64d72e36d704592ce2a614a2a2c2eeb4b2ff/src/index.html#L47) as well?

Optional: to make our lives easier, we could set the date in code and use it to say when the banner and page content should disappear. We could even set it as a field in the YAML file, so it's a global variable we don't forget about.

Reply:
We’re removing the homepage banner once the East Suffolk alert goes out. We’ll probably bring it back a week before Reading. Would support making it more automated at that point.